app/models/reports/region_summary_schema.rb

```diff
@@ -350,9 +354,9 @@ def visited_without_bp_taken(with_ltfu: false)
 
     memoize def diabetes_patients_with_bs_taken
```

Comment:
[Naming] Let's prefix the methods also with `adjusted_`?

Reply:
I think we shouldn't because we don't prefix `adjusted_` to any of the `region_summary_schema` methods

Reply:
But there is  specific meaning to `adjusted_`, i.e., patients registered 3 or more months earlier. 
If `diabetes_patients_with_bs_taken` is a count of patients registered 3 or more months earlier, it should be prefixed with `adjusted_`